Assets/LeapMotionModules/InteractionEngine/Scripts/InteractionBehaviour.cs

```diff
@@ -297,6 +315,13 @@ public abstract class InteractionBehaviour : MonoBehaviour {
 
       _isRegisteredWithManager = true;
       _manager.RegisterInteractionBehaviour(this);
+
+      _rigidbody = GetComponent<Rigidbody>();
+      if (_rigidbody)
+      {
+        _rigidbodyHadUseGravity = _rigidbody.useGravity;
+        _rigidbody.useGravity = false;
```

Comment:
Is this because bullet is going to be accounting for the force of gravity?  If so we will need a way to set the bullet gravity force so that it matches the one in Unity.